docs/dev/PluginCredentialsProvider.md

```diff
@@ -118,7 +121,7 @@ Once it is set, we can configure storage plugin to use this way of obtaining cre
 }
 ```
 
-`secretPath` property specifies the Vault key value from which to read
+`secretPath` property specifies the Vault key value from which to read. If the plugin's `authMode` is set to `user_translation` then the `secretPath` may include a variable named `$user` which will be replaced with the Drill query username at query execution time.
```

Comment:
NIT/Question:  If `user_translation` mode is enabled, isn't it a requirement that the `secretPath` contain `$user`?

Reply:
@cgivre it's still optional and you could deliberately omit it as a trick to make user_translation translate to a single shared set of creds. To me me it seemed okay to allow that...